doc/source/development/extending.rst

```diff
@@ -251,6 +251,48 @@ To use a test, subclass it:
 See https://github.com/pandas-dev/pandas/blob/master/pandas/tests/extension/base/__init__.py
 for a list of all the tests available.
 
+.. _extending.extension.arrow:
+
+Compatibility with Apache Arrow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An ``ExtensionArray`` can support conversion to / from ``pyarrow`` arrays
+(and thus support for example serialization to the Parquet file format)
+by implementing two methods: ``ExtensionArray.__arrow_array__`` and
```

Comment:
Are these in our API docs? Maybe link to them with :meth:...

Reply:
They are not. And I also can't easily add them, as the methods are not implemented on the base class (for the reason that pyarrow right now checks for the presence of this method, and if present assuming it will work (meaning: it will raise the error raised by this method, so I rather not add this method in the base class with a NotImplementedError)).